lib/components/RoundButton/RoundButton.tsx

```diff
@@ -0,0 +1,53 @@
+import { useId } from 'react';
+import classNames from 'classnames';
+
+export interface RoundButtonProps {
+  /** Text shown on the button */
+  label: string;
+  /** Callback fired on tap/click of the button */
+  onClick: () => void;
+  /** Button disabled for any actions */
+  disabled?: boolean;
+  /** Selected */
+  selected?: boolean;
+  /** CSS classname */
+  className?: string;
+}
+
+export const RoundButton = ({ label, onClick, disabled = false, selected = false, className }: RoundButtonProps) => {
+  const labelId = useId();
+  return (
+    <div
+      className={classNames('jod-roundbutton', className, 'flex flex-col items-center justify-center px-1', {
+        'cursor-not-allowed opacity-50': disabled,
+      })}
+    >
+      <button
+        aria-labelledby={labelId}
+        disabled={disabled}
+        type="button"
+        onClick={onClick}
+        className={classNames(
+          'size-[72px] rounded-full border border-transparent  hover:ring hover:ring-purple-500 focus:outline-none focus:ring focus:ring-purple-500 focus:hover:border-none',
+          {
+            'bg-[#697077]': selected,
+            'bg-[#f5f5f5]': !selected,
+          },
+        )}
+      >
+        {/** TODO: Need real icons and alignment and styles to be adjusted then */}
+        <span
+          className={classNames('jod-roundbutton-icon', 'text-center text-5xl leading-6', {
+            'text-[#4d5358]': !selected,
+            'text-[#ffffff]': selected,
+          })}
+        >
+          &#9776;
+        </span>
+      </button>
```

Comment:
If the label is part of the button, then `<button />` could maybe be the root element. E.g.
```html
<button>
  <span>&#9776;</span>
  <span>{label}</span>
<button>
```
The button could reserve some space at the bottom and the label could use absolute position to move it outside the button.

At least in this way the DOM structure could be optimized a bit. What are your thoughts on this?

Reply:
No strong opinions here, we can make the button root element if that helps.
Seemed nice for building the focus/hover ring logic to have the label text out of this. But as the hover/focus ring designs are not yet known, it does not actually matter where it goes.

Reply:
True. Focus / Hover / outline behaviour is not known yet. We can use this structure for now.

Reply:
Let the label be outside of the button for now, but we can refactor it later if need to be.